streampipes-data-explorer-commons/src/main/java/org/apache/streampipes/dataexplorer/commons/influx/InfluxStore.java

```diff
@@ -50,20 +50,32 @@ public class InfluxStore {
 
   private RawFieldSerializer rawFieldSerializer = new RawFieldSerializer();
 
-  public InfluxStore(DataLakeMeasure measure,
-                     InfluxConnectionSettings settings) {
+  /**
```

Comment:
Can we get rid of this testing-only constructor?
E.g., by using Iflux connection provider?

Reply:
Ok, thanks for the feedback and the suggestion.
I will try my best ;)